### PR TITLE
added unit tests in user.py and factory.py increase coverage

### PR DESCRIPTION
**Pull request by dcdarrell9**
### What is the context of this PR?
- Added 2 simple unit tests in user.py and factory.py to increase coverage.

### How to review 
- Check all unit tests/lint work.
- Suggest improvements?

### CODE REVIEW

File: tests/app/authentication/test_user.py [hunk 1]
```diff
@@ -22,3 +22,8 @@ def test_get_user_ik(self):
         with self.application.test_request_context():
             user = User("1", "2")
             self.assertEqual("2", user.user_ik)
+
+    def test_no_user(self):
+        with self.application.test_request_context():
+            with self.assertRaises(ValueError):
+                User("", "")
```
Comment (collisdigital) [inline tests/app/authentication/test_user.py:34]: Worth a test passing None as well? And below...

Comment (dcdarrell9) [reply tests/app/authentication/test_user.py:34]: Added...
Just a question, with this `if user_id and user_ik:` 
Is it possible to have one and not have the other, or do they both have to be there?
By this I mean in the app, not in this logic

Comment (collisdigital) [reply tests/app/authentication/test_user.py:34]: Both should be mandatory

File: tests/app/utilities/test_factory.py [hunk 2]
```diff
@@ -32,5 +32,10 @@ def test_register_all(self):
         self.assertIsInstance(factory.create("test-a"), TestClassA)
         self.assertIsInstance(factory.create("test-b"), TestClassB)
 
+    def test_create_value_error(self):
```
Comment (jonnyshaw89) [inline tests/app/utilities/test_factory.py]: I can't find anywhere that we actually use `Factory` within the code.
Maybe it should just be removed?

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: tests/app/utilities/test_factory.py (hunk 2 above)
Comment (dcdarrell9) [reply tests/app/utilities/test_factory.py]: I think you're right, I can't seem to find it being used, 'SubmitterFactory' is used but that's a different thing.

**Review by jonnyshaw89 — APPROVED** [review]

**Pull request merged**